cases/deepkit/tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "module": "commonjs",
-    "outDir": "build"
+    "module": "ESNext",
+    "outDir": "build",
+    "allowImportingTsExtensions": false
```

Comment:
Please explain what these changes do

Reply:
Since we inherited the tsconfig we need to flip the allowImportingTsExtensions setting, because it won’t work when transpiling, only for generating types. https://www.typescriptlang.org/tsconfig/#allowImportingTsExtensions

The other two changes here are to make this module ESM to match the top-level module, since we can’t mix commons/ESM.